src/app/actions/entry/getNextEntry.js

```diff
@@ -0,0 +1,32 @@
+'use server';
+import { cookies } from 'next/headers';
+import redis from '@/lib/config/redis.js';
+
+import { connectToDatabase } from '@/lib/config/postgres.js';
+const { pool } = connectToDatabase();
+
+export const getNextEntry = async (currentEntryDate) => {
+  const heliosAuth = cookies().get('heliosAuth')?.value;
+  if (!heliosAuth) return { code: 401 };
+  const travelerId = await redis.hget(`heliosTraveler:${heliosAuth}`, 'travelerId');
+  if (!travelerId) return { code: 440 };
+
+  const query = `
+      SELECT *
+      FROM entry
+      WHERE created_at::DATE > $1::DATE
+      ORDER BY created_at::DATE ASC
+      LIMIT 1;
+    `;
```

Comment:
nit: prefer to keep the query const closer to the query on line 24. While in theory this will make this func more memory efficient, its mostly so we can grok the source easier especially when things start to get more complicated